app/assets/stylesheets/_errors.scss

```diff
@@ -9,5 +9,22 @@ form {
     font-style: italic;
     color: $red;
     margin-top: 5px;
+    visibility: hidden;
```

Comment:
I'm not sure it's a good idea to add `visibility: hidden` by default to any `.error`. This might unintentionally hide errors in some places. We would need to make sure `.error` isn't used somewhere without `.field_with_errors` when it's supposed to show.
In any case, this doesn't seem to be related to this change, right? So I'd rather move it to a separate PR.

Reply:
To make the name less general i rename it to `error-field`

Reply:
But now you're still changing an existing style defintion which will impact existing references to that.
And some of these style changes do not seem to be relevant for fixing the issue for validation of institution name.

So I'd suggest to remove all the unrelated changes, such as the visibility, and introduce them in a separate PR that should deal with standardizing form markup and styles all over the app.

Reply:
Yes, please keep `.error` for the time being, with a note that it's deprecated!

Reply:
If I rollback to `.error` then I have to keep the visibility rule, since another rule in `_forms.scss`

```
div + .error,
input + .error {
  visibility: hidden;
}

div.input-required + .error,
input.input-required + .error {
  visibility: visible;
}
```
keep it hidden. So I leave it with a different class name and no visibility rule or I keep the `error` class with the visibility rule to make it work.

Reply:
Let's leave it like that and refactor these error styles in a follow-up.